crates/goose/src/prompts/system.md

```diff
@@ -51,7 +51,7 @@ No extensions are defined. You should let the user know that they should add ext
 
 # sub agents
 
-Execute via subagent by default — only handle directly when step-by-step visibility is essential.
+Execute via subagent when appropriate — handle directly when step-by-step visibility is essential.
```

Comment:
shouldn't we just replace this with a sentence that says what appropriate is? 

Execute self contained tasks where step-by-step visibility is not important through subagents